modules/core/src/test/java/org/apache/ignite/logger/java/JavaLoggerFormatterTest.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2023 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.logger.java;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+import org.apache.ignite.testframework.junits.common.GridCommonTest;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Java log formatter test.
+ */
+@GridCommonTest(group = "Logger")
+public class JavaLoggerFormatterTest {
+
+    private static JavaLoggerFormatter formatter;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        formatter = new JavaLoggerFormatter();
```

Comment:
what's the point of having this method? `formatter` field can be initialized directly in the declaration

Reply:
Agree, made out of habit